plugins/modules/eos_ospfv2.py

```diff
@@ -0,0 +1,1458 @@
+                  summary_lsa:
+                    description: Override summary-lsa metric with max-metric value.
+                    type: dict
+                    suboptions:
+                        set:
+                            description:
+                                - Set external-lsa attribute.
+                            type: bool
+                        max_metric_value:
+                            description:
+                                - Set max metric value for external LSAs.
+                            type: int
+          log_adjacency_changes:
+            description: To configure link-state changes and transitions of OSPFv2 neighbors.
+            type: dict
+            suboptions:
+              detail:
+                description: If true , configures the switch to log all link-state changes.
+                type: bool
+          maximum_paths:
+            description: Maximum number of next-hops in an ECMP route.
+            type: int
+          mpls_ldp:
+            description: mpls ldp sync configuration.
+            type: bool
+          networks:
+            description: Configure routing for a network.
+            type: list
+            elements: dict
+            suboptions:
+              network_address:
+                description: Network Address.
+                type: str
+              prefix:
+                description: Prefix.
+                type: str
+              mask:
+                description: Network Wildcard Mask.
+                type: str
+              area:
+                description: Configure OSPF area.
+                type: str
+          passive_interface:
+            description: Include interface but without actively running OSPF.
+            type: dict
+            suboption:
+              interface_list:
+                description: Interface range.
+                type: str
+              default:
+                description: If True, Set all interfaces to passive by default
+                type: bool
+          point_to_point:
+            description:  Configure Point-to-point specific features.
+            type: bool
+          rfc1583compatibility:
+            description: Specifies different methods for calculating summary route metrics. 
+            type: bool
+          distance:
+            description: Specifies the administrative distance for routes.
+            type: dict
+            suboptions:
+              external:
+                description: Routes external to the area
+                type: int
+              inter_area:
+                description: Routes from other areas
+                type: int
+              intra_area:
+                description: Routes with in an area
+                type: int
+          redistribute:
+            description: Specifies the routes to be redistributed
+            type: list
+            elements: dict
+            suboptions:
+              routes:
```

Comment:
I have used `protocol` to identify this. Wdyt?

Reply:
I prefer to have it as 'routes' . Options like 'connected' and 'static' are not protocols. redistribute command actually redistribute routes. So I think 'routes' will make sense.